{"uuid": "c014b6a6-4d6d-4d90-b193-44be4f317ddb", "children": ["ff3f1968-3ca2-4798-af07-4731b9b54c88", "6bf754a2-81d7-4b1d-8fd6-115d7b931a3b", "2c4136a0-ca95-4e3f-8a9d-b14b79818c13", "c38dda9e-697d-44fb-a47d-1c11ad38eba1", "6e9cdb11-7e99-4f2b-b3f8-764e9efc65d9", "abe80bac-7c3e-4070-acf2-f9f40211b9d3"], "befores": [{"name": "setup_profiles", "status": "broken", "statusDetails": {"message": "KeyError: 'wlan0-1'\n", "trace": "  File \"/usr/local/lib/python3.8/site-packages/pluggy/callers.py\", line 187, in _multicall\n    res = hook_impl.function(*args)\n  File \"/usr/local/lib/python3.8/site-packages/_pytest/fixtures.py\", line 1126, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n  File \"/usr/local/lib/python3.8/site-packages/_pytest/fixtures.py\", line 925, in call_fixture_func\n    fixture_result = next(generator)\n  File \"/wlan-testing/tests/e2e/basic/conftest.py\", line 79, in setup_profiles\n    return_var = fixtures_ver.setup_profiles(request, param, setup_controller, testbed, get_equipment_id,\n  File \"/wlan-testing/tests/fixtures_2x.py\", line 292, in setup_profiles\n    ap_wifi_data = ap_ssh.get_interface_details()\n  File \"/wlan-testing/tests/../libs/apnos/apnos.py\", line 408, in get_interface_details\n    wifi_info[i].append(data[i])\n"}, "attachments": [{"name": "ucentral_config: ", "source": "f7b20d6d-4736-43be-af9c-a12a6da11c31-attachment.attach"}, {"name": "Time Took to apply Config: 34.17705750465393", "source": "da545e9d-5ecc-44b8-80fb-4c8a657fb68a-attachment.attach"}, {"name": "Success : Active Config in AP: ", "source": "4d500e84-1456-4db8-a07e-d413bc36f2b0-attachment.attach"}, {"name": "AP LOgs: ", "source": "6ba7243e-7dd2-4c27-91c0-e3d5b826e50c-attachment.attach"}], "start": 1629505761447, "stop": 1629505828087}], "start": 1629505761447, "stop": 1629505828502}